service/src/main/java/edu/tamu/catalog/controller/CatalogAccessController.java

```diff
@@ -64,6 +68,24 @@ public ApiResponse getHolding(@RequestParam(value="catalogName", defaultValue="e
         }
     }
 
+    /**
+     * Provides data for all fees and fines associated with a patron.
+     *
+     * @param String catalogName (optional)
+     * @param String user
+     * @return
+     */
+    @GetMapping("/fines")
+    public ApiResponse getHolding(@RequestParam(value="catalogName", defaultValue="folio") String catalogName, @WeaverUser User user) {
```

Comment:
We really do not want to return ApiResponse. This is a bad practice due to a previous agnostic endpoint design.